db/test/test_similarity.py

```diff
@@ -0,0 +1,120 @@
+import copy
+import json
+import os.path
+
+import mock
+
+import db
+import db.similarity
+import db.exceptions
+from db.testing import DatabaseTestCase, TEST_DATA_PATH, gid_types
+
+
+class SimilarityDBTestCase(DatabaseTestCase):
```

Comment:
I'll fix the tests in here once we get some of the other things in this review solidified!